### ISSUE TITLE
dose official ffmpeg support tensorrt dnn backend?

### ISSUE THREAD

**Issue by Feynman1999** [DutchPiPi/FFmpeg-trt-backend-test#1]
dose official ffmpeg support tensorrt dnn backend?

**Comment by DutchPiPi** [comment]
No, tensorrt offers c++ API and it's hard to persuade the community to accept c++ code in ffmpeg. But we do have a tensorrt filter in GMAT: https://github.com/NVIDIA/GMAT/blob/main/ffmpeg-gpu/libavfilter/vf_tensorrt.c.
